src/app/app.go

```diff
@@ -0,0 +1,32 @@
+package app
+
+import (
+	"fmt"
+	"github.com/spf13/viper"
+	"golang-bootcamp-2020/controllers"
+	"golang-bootcamp-2020/infrastructure/router"
+	"golang-bootcamp-2020/repository/db"
+	"golang-bootcamp-2020/repository/rest"
+	"golang-bootcamp-2020/services"
+)
+
+//Setting application
```

Comment:
By convention, the descriptions should start with the struct, interface, function, etc, name:
`// StartApp - Setting application`